gulpfile.js

```diff
@@ -57,31 +71,61 @@ const cleanDist = (cb) => {
 };
 cleanDist.displayName = '🧹 cleaning up dist path';
 
+/**
+ * Copy Sass layout files from design-system-tokens to theme styles folder
+ */
+
+const copySassLayoutTokens = (cb) => {
+  const sassLayoutTokensFiles = `${tokensPackageFiles}/css-vars/${theme()}-layout-tokens.scss`;
```

Comment:
Okay, I see that it's because it's related to the generation of the css-vars. Keeping it here is a good reminder that we haven't completely become independent of Sass variables.